launch_client_gameliftlocal.cfg

```diff
@@ -0,0 +1,2 @@
+r_displayInfo 3
```

Comment:
display info 3 not required, right?

Reply:
I just replicated [what's in launch_server.cfg](https://github.com/o3de/o3de-multiplayersample/blob/development/launch_server.cfg#L1)
(sv_multithreadedConnectionUpdates = true wasn't there when this was forked)